.circleci/config.yml

```diff
@@ -273,13 +273,13 @@ commands:
           name: install podman
           command: |
             sudo mkdir -p /etc/apt/keyrings
-            curl -fsSL https://download.opensuse.org/repositories/devel:kubic:libcontainers:unstable/xUbuntu_$(lsb_release -rs)/Release.key \
+            curl -fsSL https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/unstable/xUbuntu_$(lsb_release -rs)/Release.key \
               | gpg --dearmor \
               | sudo tee /etc/apt/keyrings/devel_kubic_libcontainers_unstable.gpg > /dev/null
             echo \
               "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/devel_kubic_libcontainers_unstable.gpg]\
-                https://download.opensuse.org/repositories/devel:kubic:libcontainers:unstable/xUbuntu_$(lsb_release -rs)/ /" \
-              | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:unstable.list > /dev/null
+                https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/unstable/xUbuntu_$(lsb_release -rs)/ /" \
+              | sudo tee /etc/apt/sources.list.d/devel:/kubic:/libcontainers:/unstable.list > /dev/null
```

Comment:
@fgiorgetti I've checked that this curl works:
`https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/unstable/xUbuntu_22.04/Release.key`
but can you double check if the line 282 is correct?

Reply:
it is failing in circle ci, I have found this error while creating the release: https://app.circleci.com/pipelines/github/skupperproject/skupper/2491/workflows/541fb4f0-a9e2-4cd6-a428-532e883e93f3/jobs/13524/parallel-runs/0/steps/0-118

Reply:
@fgiorgetti  fyi: restored the 282 line